### ISSUE TITLE
Add Protective Gear to the Chosen

### ISSUE THREAD

**Issue by ChaelCodes** [ChaelCodes/HuntersKeepers#263]
## Describe the Feature
The Chosen has access to protective gear, and this is not reflected in the seedfile, because they typically have custom weapons. This issue is just to create the protective gear.

**Additional context**
> You can have protective gear worth 1-armour, if you want.

Quote from the book.

## Describe the Technical Implementation of the Solution
**Solution**
Add one piece of Gear to the [chosen seed file](https://github.com/ChaelCodes/HuntersKeepers/blob/main/db/seeds/playbooks/chosen.seeds.rb).

Attributes | Value
--- | ---
armor | 1
description | Protective Gear
harm | 0
name | Protective Gear
playbook_id | <the id of the chosen>

**References**

- [Rules](https://www.evilhat.com/home/wp-content/uploads/2019/03/Monster-of-the-Week-Revised-Playbooks.pdf)
- [Seedfiles Documentation](https://github.com/ChaelCodes/HuntersKeepers/wiki/Seedfiles)

**Comment by jonsugar** [comment]
As requested, commenting for clarifications. 

So, are you literally asking for just the seed? ... I have found [20191215140939_create_gears.rb](https://github.com/ChaelCodes/HuntersKeepers/blob/main/db/migrate/20191215140939_create_gears.rb) and [20191215141222_create_hunters_gear.rb](https://github.com/ChaelCodes/HuntersKeepers/blob/main/db/migrate/20191215141222_create_hunters_gear.rb) so I think I just need to add it around line 90 on [chosen.seeds.rb](https://github.com/ChaelCodes/HuntersKeepers/blob/main/db/seeds/playbooks/chosen.seeds.rb).

**Comment by ChaelCodes** [comment]
Yes! It actually is this simple. Asking a newbie to build an entire feature according to the standards of a codebase is a bit much in my opinion. There are a few issues in the list that are meaty like that, but they're not labeled "Good First Issue", at least, I hope they aren't!

Meaty issues like that are common in Rails projects as well, but in my experience, attempting to chop it up finer really helps in terms of delivery! Code Reviews of large features take a lot longer too.
